src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Compiler/PropertyInfoPassTest.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Symfony\Bundle\FrameworkBundle\Tests\DependencyInjection\Compiler;
 
+use Prophecy\Argument;
```

Comment:
There is no need to add prophecy to `composer.json` as it is included in PHPUnit since 4.5. We use PHPUnit and Prophecy is a part of PHPUnit. Sebastian Bergmann even recommend to use Prophecy with PHPunit instead of the old system: https://thephp.cc/news/2015/02/phpunit-4-5-and-prophecy
Why should we avoid to use it?

Btw I've used prophecy here because the test is more accurate than with old mocks. I test that all methods of `Definition` and `ContainerBuilder` that must be called are really called. I don't know if it's even possible to that using old PHPUnit mocks.